hack/e2e.go

```diff
@@ -578,6 +571,131 @@ func (k kops) Down() error {
 	return finishRunning("kops delete", exec.Command(k.path, "delete", "cluster", k.cluster, "--yes"))
 }
 
+type KubernetesAnywhere struct {
+	Path           string
```

Comment:
Same with these. Fields should start unexported and be made public only when needed.